electra.yaml

```diff
@@ -0,0 +1,21 @@
+participants:
+  - el_type: ethereumjs
+    el_image: ethereumjs:local
+    cl_type: grandine
+    cl_image: ethpandaops/grandine:feature-electra
```

Comment:
Doesn't seem like that should belong to this PR?

Reply:
It doesn't.  Accidentally included and will remove.